Cargo.toml

```diff
@@ -103,3 +103,7 @@ targets = [
     "x86_64-apple-darwin",
     "x86_64-pc-windows-msvc",
 ]
+
+# Patch the rustix dependency in is-terminal.
+[patch.crates-io]
+rustix = { path = "." }
```

Comment:
Can `Cargo.toml` uploaded to crates.io use `patch` though by default?

Reply:
Oh, good point.

Actually, I think using is-terminal here is the mistake. We're effectively testing against our own implementation. I'll pull this part of the patch, and submit a different PR to fix this.

Reply:
Yeah, I think we probably need to do a reverse-dependency style test for is-terminal or so.